README.md

```diff
@@ -108,35 +108,6 @@ Now let's talk to this node. The easiest way to do that is to use the `cockroach
 
 Check out `./cockroach help` to see all available commands.
 
-#### Building the Docker images yourself
-See [build/README.md](build/) for more information on the available Docker
-images `cockroachdb/cockroach` and `cockroachdb/cockroach-dev`.
-You can build both of these images yourself:
-
-* `cockroachdb/cockroach-dev`: `(cd build ; ./build-docker-dev.sh)`
-* `cockroachdb/cockroach`: `(cd build ; ./build-docker-deploy.sh)`
```

Comment:
yeah, it exists, but I don't think we still want to point users at it; we've had some newcomers get tripped up on this docker stuff, so i think having folks treat cockroach like any other go project is going to be a nice simplification